opentreemap/treemap/models.py

```diff
@@ -1230,6 +1230,7 @@ class Boundary(models.Model):
                                       db_index=True)
 
     canopy_percent = models.FloatField(null=True)
+    searchable = models.NullBooleanField()
```

Comment:
I think we still want a `default=True` here so that the migration will fill the column with `'t'`s for us.